src/EditorFeatures/Core/IDebuggerTextView.cs

```diff
@@ -11,8 +11,6 @@ internal interface IDebuggerTextView
     {
         bool IsImmediateWindow { get; }
 
-        void HACK_StartCompletionSession(IIntellisenseSession editorSessionOpt);
```

Comment:
This references IIntellisenseSession, so I split this interface in 2 and moved this part to EditorFeatures.Wpf